components/post_view/post_message_view/post_message_view.jsx

```diff
@@ -173,33 +184,50 @@ export default class PostMessageView extends React.PureComponent {
             {
                 shouldProcessNode: () => true,
                 processNode: processNodeDefinitions.processDefaultNode
+            },
+            {
+                shouldProcessNode: () => true,
```

Comment:
Looks like this is being done twice since it's duplicated from above

Reply:
Yeah, forgot to remove this when I initially intended to have the logic on that section.